### DISCUSSION TITLE
이미 읽은 메시지인데도 메시지 아이콘에 신규알림이 반복적으로 보여요.

### DISCUSSION THREAD

**Discussion by elliemina** [metamong-friends/metamong-friends.github.io#56] [제보하기]
우선, 메타몽 멋지네요!

- 조건: Lobby > WorkingZone1에 진입했을 때 메시지 아이콘에 신규알림 표시가 되어 메세지를 읽었는데,
- 상황: WorkingZone1 밖으로 나갔다가 다시 진입했을 때 
- 예상: 메시지 아이콘에 신규알림 표시가 없다.
- 결과: 메시지 아이콘에 신규알림이 표시된다.

![image](https://github.com/metamong-friends/metamong-friends.github.io/assets/84356996/480afac4-0ef1-480a-94aa-a35b13185ee8)

**Comment by ACT-TJ** [comment]
안녕하세요!

ACT Beta 첫 제보를 해주셔서 감사드립니다!! 
말씀하신 대로라면 Bug 로 보이는데요; Bug로 분류해
확인해 처리할 수 있도록 하겠습니다.

처리가 되면 해당 discussion 에 답변 및 closed 처리가 되며,
릴리즈 노트에 해당 내용이 포함됩니다.

릴리즈에 대한 업데이트는 실행 시 자동으로 이루어 집니다. 참고 부탁드릴게요!
